src/it/scala/com/exasol/cloudetl/kafka/KafkaConsumerPropertiesIT.scala

```diff
@@ -1,12 +1,14 @@
 package com.exasol.cloudetl.kafka
 
+import com.exasol.ExaMetadata
+
 import org.apache.avro.generic.GenericRecord
 import org.apache.kafka.clients.consumer.KafkaConsumer
 
 class KafkaConsumerPropertiesIT extends KafkaIntegrationTest {
 
   test("build returns a KafkaConsumer[String, GenericRecord]") {
-    val kafkaConsumer = KafkaConsumerProperties(properties).build()
+    val kafkaConsumer = KafkaConsumerProperties(properties).build(mock[ExaMetadata]))
```

Comment:
```suggestion
    val kafkaConsumer = KafkaConsumerProperties(properties).build(mock[ExaMetadata])
```

Reply:
This was the issue. Please pull commit and pull this change. Or you can also locally change and push.